CHANGELOG.md

```diff
@@ -1,7 +1,44 @@
+# 4.0.0 (2020-05-12)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#53](https://github.com/blackbaud/skyux-http/pull/53)
+
+### Breaking changes:
+- Removed `SkyAuthHttp`. Please use Angular's `HttpClient` instead, and import the `SkyAuthHttpClientModule` in order to implement features such as Blackbaud authentication. [#46](https://github.com/blackbaud/skyux-http/pull/46)
```

Comment:
I'd remove "Please" and start with "Instead, use...". I'd also remove "in order" and just leave it as "import the `SkyAuthHttpClientModule` to implement...".